components/TodoList.tsx

```diff
@@ -13,15 +14,32 @@ function EmptyTodo() {
 const ListTodos = observer(function ListTodos() {
   const store = useContext(TodoContext)!;
 
-  return store.todos.map((todo) => (
-    <ListItem
-      key={todo.id}
-      title={todo.text}
-      borderColor="black"
-      borderWidth="2px"
-      borderRadius="1"
-    />
-  ));
+  return (
+    <Accordion type="multiple">
+      {store.todos.map((todo) => (
```

Comment:
If this is going to be used for native platforms also you should use a 'FlatList' instead of just map.
If it's just web, then it's OK.

Reply:
Shouldn't accordion deal with this itself?

Reply:
It should, but I'm not 100% it does.
It can show with large number of items.
For the time being, it should be OK.